src/PAModel/CanvasDocument.cs

```diff
@@ -445,6 +445,20 @@ private void StabilizeAssetFilePaths()
                 if (!_assetFiles.TryGetValue(assetFilePath, out var fileEntry))
                     continue;
 
+                if (_entropy.LocalResourceFileNames.ContainsKey(resource.Name))
```

Comment:
The order of items in the resources array in Resources.json appears to be random. 
Which means our traversal here could be random. 
Which means we may randomly decide which resource gets the "image.jpg" and which faults over to "image_1".jpg. 
Which could lead to noisy diffs. 

Probably easiest fix is to sort the traversal on line 439.